### PR TITLE
Paragraph templates

### PR DESCRIPTION
**Pull request by yepzdk**
#### Link to ticket

https://leantime.itkdev.dk/dashboard/home?tab=ticketdetails#/tickets/showTicket/1446
https://leantime.itkdev.dk/dashboard/home?tab=timesheet#/tickets/showTicket/1444
https://leantime.itkdev.dk/dashboard/home?tab=timesheet#/tickets/showTicket/1443
https://leantime.itkdev.dk/dashboard/home?tab=timesheet#/tickets/showTicket/1440

#### Description

Create paragraphs:
- Hero
- Introduction
- Content list
- Faq
- Cta

#### Screenshot of the result

Screenshot 2024-06-11 at 14 39 17
Screenshot 2024-06-06 at 08 43 26
Screenshot 2024-06-11 at 11 19 38
Screenshot 2024-06-11 at 11 19 30
Screenshot 2024-06-11 at 11 19 20

#### Checklist

- [ ] My code is covered by test cases.
- [ ] My code passes our test (all our tests).
- [ ] My code passes our static analysis suite.
- [ ] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist, you have to add a comment explaining why this change
should be exempt from the list.

#### Additional comments or questions

If you have any further comments or questions for the reviewer, please add them here.

### CODE REVIEW

File: web/themes/custom/ai_launchpad/templates/chat/chat-window.html.twig [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% set classes = ['chat-window border p-4 rounded', ui.buttons ? ' fixed bottom-0 right-0 bg-primary w-full md:w-3/5 lg:w-2/5 min-w-[470px]' : 'my-4 max-w-3xl'] %}
+{% set classes = ['chat-window border border-secondary p-4 rounded', ui.buttons ? 'hidden fixed bottom-0 right-0 bg-primary w-full md:w-3/5 lg:w-2/5 min-w-[470px] z-50' : 'my-4 max-w-3xl'] %}
```
Comment (cableman) [inline web/themes/custom/ai_launchpad/templates/chat/chat-window.html.twig:1]: Wow

File: web/themes/custom/ai_launchpad/templates/paragraphs/paragraph--call-to-action.html.twig [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{% block paragraph %}
+  <section class="my-9 md:my-24 lg:my-32 px-3 text-center relative">
+    {% block content %}
+      <div class="max-w-lg mx-auto z-10">
+        <h1 class="text-4xl font-semibold mx-4 text-balance">{{ elements.field_title.0['#context'].value }}</h1>
```
Comment (cableman) [inline web/themes/custom/ai_launchpad/templates/paragraphs/paragraph--call-to-action.html.twig:5]: Do twig not have some helper to get first value? And not the 0 index?

**Review by cableman — APPROVED** [review]

File: web/themes/custom/ai_launchpad/templates/chat/chat-window.html.twig (hunk 1 above)
Comment (yepzdk) [reply web/themes/custom/ai_launchpad/templates/chat/chat-window.html.twig:1]: Responsive width control..

File: web/themes/custom/ai_launchpad/templates/paragraphs/paragraph--call-to-action.html.twig (hunk 2 above)
Comment (yepzdk) [reply web/themes/custom/ai_launchpad/templates/paragraphs/paragraph--call-to-action.html.twig]: Unfortunatly nothing that I know off.

**Pull request merged**